src/video/wayland/SDL_waylandevents_c.h

```diff
@@ -99,6 +99,9 @@ struct SDL_WaylandInput
     struct wl_seat *seat;
     struct wl_pointer *pointer;
     struct wl_touch *touch;
+    struct zwp_pointer_gestures_v1 *pointer_gestures;
+    struct zwp_pointer_gesture_pinch_v1 *gesture_pinch;
+    struct zwp_pointer_gesture_swipe_v1 *gesture_swipe;
```

Comment:
The swipe gesture here seems to be unused.

Reply:
yes done